### DISCUSSION TITLE
Microdot mentioned in Python Bytes #330 podcast

### DISCUSSION THREAD

**Discussion by mattytrentini** [miguelgrinberg/microdot#125] [General]
I thought some folks would be interested to hear Microdot discussed in the recent [Python Bytes #330](https://pythonbytes.fm/episodes/show/330/your-data-validated-5x-50x-faster-coming-soon) podcast by host Michael Kennedy at about 8:18. Mike seems to like it. ☺️

**Comment by miguelgrinberg** [comment]
I appreciate that Mike even created two identical hello world type apps in Flask and Microdot and compared their memory footprint. :)
